issues-workflow.md

```diff
@@ -27,6 +27,8 @@ Here are the custom labels introduced by the Kyma team:
 * `priority/critical` indicates the top-priority of a given issue.
 * `area/{CAPABILITY_NAME}` indicates which capabilities are related to a given issue. You can assign more than one `area` label an issue.
 * `security/{SCORE}` indicates the security issue based on the [CVSSv3](https://www.first.org/cvss/calculator/3.0) security rating.
+* `sig/{SIG_NAME}` indicates [which Special interest group](./sig-and-wg/README.md#special-interest-groups-and-working-groups) identified the issue and is responsible for further follow-up of the issue.
```

Comment:
``* `sig/{SIG_NAME}` indicates which [Special Interest Group](./sig-and-wg/README.md#special-interest-groups-and-working-groups) identified the issue and is responsible for the further follow-up on the issue.``